src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -132,11 +132,11 @@ public void setDefaultContributorRole(DataverseRole defaultContributorRole) {
     private List<DataverseContact> dataverseContacts = new ArrayList();
     
     @OneToMany(cascade = {CascadeType.MERGE})
-    private List<MetadataBlock> metadataBlocks = new ArrayList();
+    private List<MetadataBlock> metadataBlocks = new ArrayList<MetadataBlock>();
```

Comment:
Equally, we could use diamond inference here, right? Like this: `new ArrayList<>();`